models/repo.go

```diff
@@ -336,10 +336,20 @@ func (repo *Repository) UnitEnabled(tp UnitType) bool {
 	return false
 }
 
-var (
-	// ErrUnitNotExist organization does not exist
-	ErrUnitNotExist = errors.New("Unit does not exist")
-)
+// ErrUnitTypeNotExist represents a "UnitTypeNotExist" kind of error.
+type ErrUnitTypeNotExist struct {
+	UT UnitType
+}
+
+// IsErrUnitTypeNotExist checks if an error is a ErrUnitNotExist.
+func IsErrUnitTypeNotExist(err error) bool {
+	_, ok := err.(ErrUnitTypeNotExist)
+	return ok
+}
+
+func (err ErrUnitTypeNotExist) Error() string {
+	return "Unit does not exist"
```

Comment:
It's better to display which unit is not exist.

Reply:
Didn't want to change what already existed too much, just needed to be able to know it was a ErrUnitTypeNotExist err, but ok, will make that make more sense. :)

Reply:
Ok, fixed.